### ISSUE TITLE
compare old OSPREE results to phylo results

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/ospree#436]
@MoralesCastilla wrote:
I’m now writing the results of the ospree phylogeny ms. and somewhere there (or in the discussion) I would like to compare our results including the phylogeny against those from the all sps model in the NCC paper. I’ve been trying to retrieve those results but I realize I may not have the .Rda files with the model outputs from the NCC paper. If there is a way to zip those and send them to me so I do not need to run them again, that would be great!

In any case, I have a few questions in order to do these comparisons. The thing is that the all species model contains 203 species, and we are modelling angiosperms (192sps) and gymnosperms (19sps) separately and excluding crops. This means we may not get to compare exactly equivalent models. Since the goal is not to make an exhaustive comparison but check for overall consistency (i.e. correlations?) among species-level slopes from different approaches, perhaps this is not a big issue and can be circumvented by:
*    Selecting overlapping subsets of species in the NCC all sps model and our angiosperm model and checking correlations in species-level slopes (my original intent).
*   Re-running our phylo model for exactly the same species subset as in NCC all species model (it would be hard to do that for the complexes model as it would involve collapsing the phylogeny, which can lead to new issues)

Or, we can forget about the NCC results and focus in the new ones (that would be sad in my opinion, as we would miss the chance to say whether including phylogeny may shift results or not).

**Comment by lizzieinvancouver** [comment]
@AileneKane replied:

I believe that I ran these models on my old laptop, which unfortunately has died, so I don’t know that I have any .Rda files to send you- I can look at my other laptop (which is at home; I am at the office, for a change today) to see if anything is there. In any case, the models are pretty quick to run to if you have to rerun them it shouldn’t take long! I can send you the files this afternoon, if I do indeed have them.

I think your first option (“Selecting overlapping subsets of species in the NCC all sps model and our angiosperm model and checking correlations in species-level slopes”) sounds reasonable, since the goal is to compare how incorporating phylogeny alters our estimates of phenological responses.

**Comment by lizzieinvancouver** [comment]
@MoralesCastilla Hello! I just looked and I have Rda files but I would need to know the name of the one you wanted. As @AileneKane said they actually run pretty fast so maybe what you need is for @AileneKane to pull the exact code to re-run the 203 one and then you can subset down from there?

I think we should present the new data with lambda set to 0 as the best updated estimates without phylogeny, since we have more data now, but agree it would be nice to see the old estimates and compare there as well.

Thank you both for your work on this!

**Comment by AileneKane** [comment]
I agree that using the same phylogeny model with lamda set to 0 is the best way to compare effect of phylogeny, but also would be nice to see old estimates and compare to new. I can rerun the all sp (utah chilling?) model and get the .Rda to you, if that's helpful @MoralesCastilla. I'm assuming centered predictors- let me know if you need something else!
